iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/expected.inl

```diff
@@ -72,8 +64,8 @@ inline error<T>::error(Targs&&... args) noexcept
     : value(std::forward<Targs>(args)...)
 {
 }
-
-
+// AXIVION Next Construct AutosarC++19_03-A12.1.5 : Delegating ctor doesn't fit-in here, as each initialization
+// for 'value' data member is in different style
```

Comment:
A justification like `This ctor uses a rvalue reference of the underlying type which makes it similar to the move ctor and therefore a delegating ctor cannot be used` would fit better

Reply:
@elBoberido changed